### DISCUSSION TITLE
Firing at multiple position programmatically

### DISCUSSION THREAD

**Discussion by vectoron** [matteobruni/tsparticles#4931] [Q&A] [answered]
I tried going through the documentation but I am have a real challenge figuring out how to trigger multiple emitters in a fashion similar to clicking but changing the location programmatically. When I call load, it seems to stop the previous animation and start the new one where as I want to not destroy the previous on going animation in a similar fashion to click. Additionally after load is it possible to update an emitter position in real-time?

**Comment by matteobruni** [comment, marked as answer]
You can see the `addEmitter` function here in the confetti bundle for adding an emitter programmatically:

https://github.com/matteobruni/tsparticles/blob/a384765c5c48460877a4183d3f753d42ef36c14e/bundles/confetti/src/confetti.ts#L97

These are the other functions added by the emitters plugin:

https://github.com/matteobruni/tsparticles/blob/main/plugins/emitters/src/EmitterContainer.ts

**Comment by vectoron** [comment]
This is precisely what I was looking for, thank you.
